src/components/Markdown/MarkdownLink.tsx

```diff
@@ -26,6 +30,36 @@ const scrollWithTimeout = (element: any, offset: number) => {
   }, 400);
 };
 
+const CanEmbed: React.FC<{ href: string }> = ({ href }) => {
+  const [isOpen, setIsOpen] = useState(false);
+  const region =
+    href === 'https://covidactnow.org/embed/us/fips/19100'
+      ? regions.findByFipsCode('19100')!
+      : undefined;
```

Comment:
This is hacky. As the "Tools" page is not location specific, the embed preview and link which are more location specific is currently set to Texas. The way to identify which location the embed preview is referring to is by the fips code in the link in the markdown file. To ensure the embed link will show the correct embed preview box, this is currently hardcoded -__-" A better solution is welcome!